src-tauri/src/northstar/mod.rs

```diff
@@ -212,9 +214,10 @@ pub fn launch_northstar(
     {
         let ns_exe_path = format!("{}/NorthstarLauncher.exe", game_install.game_path);
         let ns_profile_arg = format!("-profile={}", game_install.profile);
+        let args_str = &launch_args.join(" ");
 
         let _output = std::process::Command::new("C:\\Windows\\System32\\cmd.exe")
-            .args(["/C", "start", "", &ns_exe_path, &ns_profile_arg])
+            .args(["/C", "start", "", &ns_exe_path, &ns_profile_arg, args_str])
```

Comment:
Isn't this what is causing the args to be interpreted as one string? you are adding them as a single argument instead of adding each argument individually

Reply:
That was exactly that :')
Addressed in 2a4fc2fbf34f5be3cdeb5031efbb17420a74cfbc